UX Changes

Leland opened the pull request:
**Note:** this is my first time contributing any React code – if it sucks, lemme know!

My Saturday project: overhauling UnwiseConnect design and UX. No new functionality here  – this PR focuses instead on (rather subjective) UX/UI changes.

The goal with this one was to simplify to a certain degree the visual hierarchy presented. I deemphasized headers, lowered the visual prominence of the navigation, and floated several elements in an effort to give greater focus to the task list.

From the UX view of a new user to the service, I also added a greeting with some simple copy to the logged out homepage and added a few notes of clarification to some lines of text. I also changed the style (e.g `btn-default` vs `-primary`) of buttons used in a few places to better distinguish the important bits of the interface.

There were also subjective design changes I made, like adding a background color, changing the font to Roboto, and moving the page contents into their own `.panel` elements.

Few other notes:

- Moved the "Logout" button from the header to the Settings page (reason being that users will rarely be logging out)
- Moved the "Login" button to the (logged out) index page's contents
- Added a simple loading spinner
- Removed navigation from logged out homepage

<img width="1206" alt="screen shot 2017-07-16 at 2 29 53 am" src="https://user-images.githubusercontent.com/714017/28245179-b6080f52-69ce-11e7-8d37-a1da7bdf8d59.png">


<img width="1220" alt="screen shot 2017-07-16 at 1 08 31 am" src="https://user-images.githubusercontent.com/714017/28244808-55afa06c-69c3-11e7-92ba-a7eb90d653c4.png">

<img width="1196" alt="screen shot 2017-07-15 at 11 59 19 pm" src="https://user-images.githubusercontent.com/714017/28244815-8eba8926-69c3-11e7-9c55-44731c78c81d.png">


![screen recording 2017-07-16 at 12 00 am](https://user-images.githubusercontent.com/714017/28244988-30cbd904-69c9-11e7-9aaf-083073afe533.gif)


<img width="300" alt="screen shot 2017-07-15 at 7 47 59 pm" src="https://user-images.githubusercontent.com/714017/28243958-2ac934bc-69aa-11e7-934b-5e2768b4b1ff.gif">

nadavspi commented on src/components/Home.js:
```diff
@@ -10,8 +10,11 @@ class Home extends Component {
-        <p>Hi.</p>
+        <div className="jumbotron">
+          <h1>UnwiseConnect</h1>
+          <hr/>
```
Old school.

Leland replied:
Jesus, tired Leland.

Leland replied:
Changed

nadavspi commented on src/components/Home.js:
```diff
@@ -14,6 +15,13 @@ class Home extends Component {
+            onClick={() => this.props.dispatch(login())}
+            className="btn btn-lg btn-primary"
+          >
+            Login with Github
```
Nit: GitHub

Leland replied:
Fixed

nadavspi commented on src/components/Settings/Toggl.js:
```diff
@@ -29,25 +29,33 @@ export default class Toggl extends Component {
+          <a
+            href="https://toggl.com/app/profile"
+            target="_blank">
+              Find your API key
```
Good idea.

nadavspi commented on src/index.css:
```diff
@@ -57,11 +57,30 @@ body {
+}
+
+.projects:hover {
+  max-height: 500px;
```
Hehe, someone will eventually have enough projects to exceed this.

Leland replied:
Removed

nadavspi commented on src/index.css:
```diff
@@ -57,11 +57,30 @@ body {
+  box-shadow: none;
+  border: solid 1px #bbbbbb;
+  margin: 0;
+  transition: max-height .95s ease-out .1s;
```
Like we talked about, let's make this (much) faster and fix the z-index issue with the toggl button.

Leland replied:
Done

Leland commented:
Killed the animation and moved the projects list to Bootstrap's list group styling. Also moved the entire project line to one button to increase the click target size.

![screen recording 2017-07-20 at 08 50 pm](https://user-images.githubusercontent.com/714017/28444698-13d24e24-6d8d-11e7-8e8a-69c28bf260ff.gif)
